### ISSUE TITLE
Add License

### ISSUE THREAD

**Issue by programmerjake** [crev-dev/crev#13]
This repo is missing a license, we should decide which license(s) to use and add them.

I suggest and give my consent for choosing `MPL-2.0 OR MIT OR Apache-2.0` as the license since that's what cargo-crev uses.

According to copyright law, all contributors are required to give their consent before adding any chosen license(s).

@dpc @JesterOrNot @jcgruenhage

**Comment by jcgruenhage** [comment]
I also give consent to that set of licenses, and any subset that includes `Apache-2.0`. ooi, have I contributed anything meaningful? ;D

**Comment by jcgruenhage** [comment]
Right, this is repo is just a README, and I contributed a link fix. In that case, treat my contributions as `CC0`, I really couldn't care less

**Comment by dpc** [comment]
Repos with code do have a license. This one is just a README

**Comment by programmerjake** [comment]
> Repos with code do have a license. This one is just a README

Text is copyrightable too.
